src/fuzz_introspector/analyses/data/cwe_data.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2023 Fuzz Introspector Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict
+
+SINK_FUNCTION: Dict[str, Dict[str, Any]] = {
+    'CWE79': {
```

Comment:
How about making this a bit more typed than a dictionary? It would be useful to have some form of metadata related to each CWE I think